host/xtest/Makefile

```diff
@@ -73,11 +73,13 @@ srcs +=	adbg/src/adbg_case.c \
 	regression_7000.c \
 	regression_8000.c \
 	regression_8100.c \
+	session_1000.c \
```

Comment:
Please move the content of this file into `regression_1000.c` instead and make it part of the regression suite instead continuing from 1024 or whatever is the next free number in the series.

Reply:
Ok.

In this case is it OK to rebase/rework the branch as it makes reviewing a bit easier? (your guidance was to add new commits and then squash/fixup at the end)

Reply:
Please go ahead and rebase and rework as needed.

Reply:
Moved, rebased and force pushed a new version